rts/System/Sound/OpenAL/SoundSource.cpp

```diff
@@ -53,6 +54,15 @@ CSoundSource::CSoundSource()
 	curPlayingItem = {0,  0, 0,  0.0f, 0.0f};
 }
 
+CSoundSource::CSoundSource(CSoundSource&& src)
```

Comment:
Can't this be `= default`?

Reply:
implemented move operations in [8b8cc67](https://github.com/beyond-all-reason/spring/pull/857/commits/8b8cc670030cc299864c60a26e56c43562bb6e4a)

Reply:
Why `swap()` and not the move assignment operator and constructor?

Reply:
> Why `swap()` and not the move assignment operator and constructor?

default move operator copies numeric `id` field which needs to be unique for each object instance